app/assets/javascripts/init.js

```diff
@@ -31,6 +31,8 @@
 
     $('#manage_flow section.applications').applicationActions();
 
+    $('#manage_flow section.deployment-details').composeExportDialog();
```

Comment:
It may be more clear to init this the same way in both cases. E.G.
```
$('#manage_flow section.deployment-details, #manage_flow section.applications').composeExportDialog();
```
And kill line 26 in jquery.application_actions.js